README.md

```diff
@@ -14,17 +14,24 @@ In this README we first focus on the workflow to update an existing template, th
 
 ## Dependencies
 
+* [Poetry](https://python-poetry.org/docs/)
 * [DoIt](http://pydoit.org/install.html)
 * [Composer](https://getcomposer.org/) (for PHP projects)
 * Git
 * Public ssh key in github account
 
 ### Setup
 
+0. Install Poetry locally
+
+Follow the installation instructions in [Poetry's documentation](https://python-poetry.org/docs/). You can verify Poetry locally with the command `poetry --version`. 
```

Comment:
Their instructions seem to be a bit old.  Following those instructions, I get a Poetry install that defaults to Python 2, gives me an ugly error, and then switches to Python 3.  Can we figure out how to have it install to Python 3 directly, and then include those instructions here?